backend/api/src/main/kotlin/software/shonk/adapters/incoming/ShorkInterpreterController.kt

```diff
@@ -55,6 +56,28 @@ fun Route.configureShorkInterpreterControllerV1() {
         program.onSuccess { call.respond(Program(it)) }
         return@get
     }
+
+    post("/lobby") {
+        val lobbyId = shorkUseCase.createLobby()
+        val result = shorkUseCase.getLobbyStatus(lobbyId)
+
+        result.onFailure {
+            call.respond(HttpStatusCode.InternalServerError, it.message ?: UNKNOWN_ERROR_MESSAGE)
+        }
+        result.onSuccess { call.respond(HttpStatusCode.Created, lobbyId.toString()) }
+        return@post
+    }
```

Comment:
sorry this was shelved by mistake from intellij since I was on the wrong branch which was @TR1NITROTOLUOL 's and I intended to shelve only my changes, but it apparently shelved his instead and I didn't notice that since I was rewriting my code